src/main/java/org/gbif/ipt/action/manage/AutoPublishAction.java

```diff
@@ -67,6 +68,9 @@ public void prepare() {
     populateBiMonths();
     populateDays();
     populateDaysOfWeek();
+
+    setServerTimeZone();
+    setUpdateFrequencyTime();
```

Comment:
This causes issues (namely, NPE) when switching off auto-publishing. After that the auto-publishing becomes inaccesible